content/en/docs/Performance/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Performance"
+weight: 150
+date: 2020-11-20
+description: >
+  Baker performance test cases
+---
+
+### Read from S3 and write to local disk
+
+On a `c5.2xlarge` instance, Baker managed to read zstandard records from S3, uncompress them and
```

Comment:
Not sure c5.2xlarge speaks to everybody, maybe we could add in parenthesis (or as a footer note as you prefer) that this is an AWS EC2 instance based on Intel Xeon Platinum 8000. Or maybe just that is an AWS instance type.
```suggestion
On a `c5.2xlarge` instance, Baker can read zstandard records from S3, uncompress them and
```